scripts/generate-package-table-for-readme.js

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env node
+
+// This script generates the table of packages you can see in the readme
+
+// Columns to show at the header of the table
+const columns = [
+  'Package',
+  'Version',
+  'Deps',
+  'DevDeps',
+  'CI',
+  'Coverage'
+]
+
+// Headings are a string
+// Arrays are packages. Index 0 is the GitHub repo and index 1 is the npm package
+const rows = [
+  'Repo',
+  ['ipfs/js-ipfs-repo', 'ipfs-repo'],
+
+  'DAG',
+  ['ipld/js-ipld', 'ipld'],
+  ['ipld/js-ipld-dag-pb', 'ipld-dag-pb'],
+  ['ipld/js-ipld-dag-cbor', 'ipld-dag-cbor'],
+
+  'Files',
+  ['ipfs/js-ipfs-unixfs-engine', 'ipfs-unixfs-engine'],
+
+  'Exchange',
+  ['ipfs/js-ipfs-block-service', 'ipfs-block-service'],
+  ['ipfs/js-ipfs-bitswap', 'ipfs-bitswap'],
+
+  'libp2p',
+  ['libp2p/js-libp2p', 'libp2p'],
+  ['libp2p/js-libp2p-circuit', 'libp2p-circuit'],
+  ['libp2p/js-libp2p-floodsub', 'libp2p-floodsub'],
+  ['libp2p/js-libp2p-kad-dht', 'libp2p-kad-dht'],
+  ['libp2p/js-libp2p-mdns', 'libp2p-mdns'],
+  ['libp2p/js-libp2p-mplex', 'libp2p-mplex'],
+  ['libp2p/js-libp2p-railing', 'libp2p-railing'],
+  ['libp2p/js-libp2p-secio', 'libp2p-secio'],
+  ['libp2p/js-libp2p-tcp', 'libp2p-tcp'],
+  ['libp2p/js-libp2p-webrtc-star', 'libp2p-webrtc-star'],
+  ['libp2p/js-libp2p-websocket-star', 'libp2p-websocket-star'],
+  ['libp2p/js-libp2p-websockets', 'libp2p-websockets'],
+
+  'Data Types',
+  ['ipfs/js-ipfs-block', 'ipfs-block'],
+  ['ipfs/js-ipfs-unixfs', 'ipfs-unixfs'],
+  ['libp2p/js-peer-id', 'peer-id'],
+  ['libp2p/js-peer-info', 'peer-info'],
+  ['multiformats/js-multiaddr', 'multiaddr'],
+  ['multiformats/js-multihash', 'multihashes'],
+
+  'Generics/Utils',
+  ['ipfs/js-ipfs-api', 'ipfs-api'],
+  ['fbaiodias/ipfs-multipart', 'ipfs-multipart'],
+  ['ipfs/is-ipfs', 'is-ipfs'],
+  ['multiformats/js-multihashing', 'multihashing'],
+  ['multiformats/js-mafmt', 'mafmt'],
+
+  'Crypto',
+  ['libp2p/js-libp2p-crypto', 'libp2p-crypto'],
+  ['libp2p/js-libp2p-keychain', 'libp2p-keychain']
+]
+
+const isItemPackage = (item) => {
+  return Array.isArray(item)
+}
+
+const packageBadges = [
+  // Package
+  (gh, npm) => `[\`${npm}\`](//github.com/${gh})`,
+  // Version
+  (gh, npm) => `[![npm](https://img.shields.io/npm/v/${npm}.svg?maxAge=86400&style=flat-square)](//github.com/${gh}/releases)`,
+  // Deps
+  (gh, npm) => `[![Dep](https://david-dm.org/${gh}.svg?style=flat-square)](https://david-dm.org/${gh})`,
+  // DevDeps
+  (gh, npm) => `[![Dep](https://david-dm.org/${gh}.svg?style=flat-square)](https://david-dm.org/${gh})`,
```

Comment:
Thinking more about this, I say let's remove DevDeps. From the consumer standpoint, only deps matter.